include/crow/routing.h

```diff
@@ -390,7 +391,8 @@ namespace crow
 
         void handle_upgrade(const request& req, response&, SocketAdaptor&& adaptor) override
         {
-            new crow::websocket::Connection<SocketAdaptor, App>(req, std::move(adaptor), app_, open_handler_, message_handler_, close_handler_, error_handler_, accept_handler_);
+            auto* conn = new crow::websocket::Connection<SocketAdaptor, App>(req, std::move(adaptor), app_, open_handler_, message_handler_, close_handler_, error_handler_, accept_handler_);
+            conn->set_max_payload_size(max_payload_);
```

Comment:
I don't know if there's a better way of doing this @The-EDev, the constructor of Connection will take the default App max payload, but right after we over write it with the Rule's max payload

Reply:
Since `handle_upgrade()` is only called when a client connects to the server, what we can do instead is have the rule take the app's `websocket_max_payload()` (which it already does), and pass it to the connection's constructor.

That way if the user calls `maxpayload()` we can simply pass the new value later when the connection object is constructed

Reply:
Updated. I wasn't sure if adding more to the constructor was the best course of action, but if you're ok with it then sure.